src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -1085,6 +1085,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				this._providerId = provider;
 			} else if (!provider) {
 				this._providerId = SQL_NOTEBOOK_PROVIDER;
+				this.notificationService?.notify({ severity: Severity.Info, message: localize('savedKernelNotSupported', "This notebook's '{0}' kernel is not supported. Defaulting to SQL kernel instead.", this._savedKernelInfo.display_name ?? this._savedKernelInfo.name) });
```

Comment:
We don't always pass in a notificationService from the unit tests. I'm temporarily adding a null conditional check here until I can take care of that service cleanup in a separate PR.